test/cli/test_cli_functions.sh

```diff
@@ -68,19 +64,14 @@ function test_noobaa {
         then
             if ${should_fail}
             then
-                echo_time "✅  ${noobaa} ${options} failed - as should"
+                echo "✅  ${noobaa} ${options} failed - as should"
             else 
-                echo_time "❌  ${noobaa} ${options} failed, Exiting"
-                local pod_operator=$(kuberun get pod | grep noobaa-operator | awk '{print $1}')
-                echo_time "==============OPERATOR LOGS============"
-                kuberun logs ${pod_operator}
-                echo_time "==============CORE LOGS============"
-                kuberun logs noobaa-core-0
```

Comment:
The print above might be helpful even after the revert

Reply:
reverted back